### ISSUE TITLE
Combination "t=squaredown w=XX h=YY" fails

### ISSUE THREAD

**Issue by kbsali** [andrieslouw/imagesweserv#178]
Hi,

we are having the following error when passing this exact combination of parameters : `&t=squaredown&w=468&h=352`

```
Error 404: Server couldn't parse the ?url= that you were looking for, error it got: libvips error: ...local/openresty/luajit/share/lua/5.1/vips/voperation.lua:245: unable to call thumbnail
smartcrop: bad extract area
```

Removing any of the 3 parameters or changing the `t` value to any other options works though.

Could it be linked to some issue in `libvips` which seems to be somewhat related?

See https://images.weserv.nl/?url=gensdeconfiance.s3-eu-west-1.amazonaws.com%2Fimages%2Fmembers%2Fthumb%2F2015_07_06_559a8a29a27ea.jpg&t=squaredown&w=468&h=352&il=1

We have not changed this combination recently, could it be related to a lib update on your side?

**Comment by kbsali** [comment]
Strange though, when using the sample image you have in your doc, it works fine : https://images.weserv.nl/?url=ory.weserv.nl/lichtenstein.jpg&w=200&h=300&t=squaredown

**Comment by ndavoust** [comment]
@kbsali it will fail if the original image is smaller than the requested dimensions:
https://images.weserv.nl/?url=ory.weserv.nl/lichtenstein.jpg&w=2000&h=2000&t=squaredown

**Comment by kbsali** [comment]
> *Will not oversample* the image if the requested size is larger than that of the original.`

you are right, and it is documented. But it still strange, I believe it used to work... :thinking:

**Comment by kbsali** [comment]
Re-opening the issue : indeed, the expected behaviour is that it should return an image no matter what combination of parameters is passed!

**Comment by kleisauke** [comment]
This seems to be a bug on our side, it should be fixed now. Conflicting line:
https://github.com/weserv/images/commit/f8dbb8870452904df805d6bf1f30da148fc3c56b#diff-52af44bb5f705fed4ea69f85a98f9b2bR162

Note that the documentation will be improved in the next API version (sneak preview here: https://t0.nl/docs/).

**Comment by kbsali** [comment]
Awesome, thanks a lot,
the bug is indeed fixed (too bad we cannot ask our users to refresh their cache!).

**Comment by bgaillard** [comment]
Hi @kleisauke and thanks a lot for your quick response.

I recently compiled a Docker container based on the last commit of the `4.x` GIT branch and encounter this error.

I'm not 100% sure to understand you when you indicate "Conflicting line". 

Is the line to be removed (I tried it locally and it seems to work removing the line) ? If yes would it be possible to update the `4.x` GIT branch with the fix ?

Thanks.
